src/pages/index.js

```diff
@@ -1,67 +1,128 @@
+  profileAbout,
+  profileAvatar,
 } from "../utils/constants.js";
-// import { create } from "lodash";
-// import { profile } from "console";
 
 // FORM VALIDATION INIT
-const profileEditFormValidator = new FormValidator(
-  validationSettings,
-  profileEditForm
-);
-const profileAddFormValidator = new FormValidator(
-  validationSettings,
-  profileAddForm
-);
+const formValidators = {};
+
+const enableValidation = (config) => {
+  const formList = Array.from(document.querySelectorAll(config.formSelector));
+  formList.forEach((formElement) => {
+    const validator = new FormValidator(config, formElement);
+    const formName = formElement.getAttribute("name");
 
-profileEditFormValidator.enableValidation();
-profileAddFormValidator.enableValidation();
+    formValidators[formName] = validator;
+    validator.enableValidation();
+  });
+};
+
+enableValidation(validationSettings);
+
+async function handleSubmit(request, popupInstance, loadingText = "Saving...") {
+  try {
+    popupInstance.renderLoading(true, loadingText);
+    await request();
+    popupInstance.close();
+  } catch (err) {
+    console.error("Error in handleSubmit:", err);
+  } finally {
+    popupInstance.renderLoading(false);
+  }
+}
 
 // USER INFO INIT
 const userInfo = new UserInfo({
   nameSelector: profileName,
-  jobSelector: profileDescription,
+  aboutSelector: profileAbout,
+  avatarSelector: profileAvatar,
 });
 
-function handleEditFormSubmit(data) {
-  userInfo.setUserInfo({
-    name: data.title,
-    job: data.description,
-  });
+let currentUserId = null;
+
+function handleEditFormSubmit(inputValues) {
+  async function makeRequest() {
+    console.log("Updating profile with values:", inputValues);
+    const userData = await api.updateProfile({
+      name: inputValues.name,
+      about: inputValues.about,
+    });
+    userInfo.setUserInfo(userData);
+  }
+  return handleSubmit(makeRequest, profileEditPopup);
+}
 
-  profileEditForm.reset();
-  profileEditFormValidator.disableButton();
-  profileEditPopup.close();
+function handleAddFormSubmit(inputValues) {
+  async function makeRequest() {
+    console.log("Adding card with values:", inputValues);
+
+    const cardData = await api.addCard(inputValues);
+    const cardElement = createCard(cardData, currentUserId);
+    cardSection.addItem(cardElement);
+  }
+  return handleSubmit(makeRequest, profileAddPopup);
 }
 
-function handleAddFormSubmit(data) {
-  const cardData = {
-    place: data.place,
-    link: data.link,
-    alt: `Image of ${data.place}`,
-  };
-  const cardElement = createCard(cardData);
-
-  cardSection.addItem(cardElement);
-  profileAddForm.reset();
-  profileAddFormValidator.disableButton();
-  profileAddPopup.close();
+function handleAvatarFormSubmit(inputValues) {
+  async function makeRequest() {
+    console.log("Updating avatar with values:", inputValues);
+    const userData = await api.updateAvatar(inputValues.link);
+    userInfo.setUserInfo(userData);
```

Comment:
* You should **clear the inputs** after a successful adding of a new card to let the user add the 2nd card again without having to remove the old data manually. 


Please, reset the form here